docs/kyma-environment-broker/03-11-swagger.md

```diff
@@ -10,10 +10,13 @@ You can either use Virtual Service or port-forward the Pod to expose and use the
 
 ## Use Virtual Service
 
-Open the following website:
+Open one of the following websites:
 
    ```
-   https://$BROKER_URL/
+   https://kyma-env-broker.cp.dev.kyma.cloud.sap/
+   ```
+   ```
+   https://kyma-env-broker.cp.stage.kyma.cloud.sap/
    ```
```

Comment:
```suggestion
- [`https://kyma-env-broker.cp.dev.kyma.cloud.sap/`](https://kyma-env-broker.cp.dev.kyma.cloud.sap/)
- [`https://kyma-env-broker.cp.stage.kyma.cloud.sap/`](https://kyma-env-broker.cp.stage.kyma.cloud.sap/)
```
There's no need to format these as full code blocks, these are just values that we list. We can, however, link them so you don't need to copy and paste them, and just click to open.